src/js/containers/generateFiles/GenerateFilesContainer.jsx

```diff
@@ -126,9 +126,7 @@ class GenerateFilesContainer extends React.Component {
 					}
 					// certified submission files have been updated, hide prepopulate D1/D2
```

Comment:
There is a merge conflict which is probably caused by the updates to this happening in a separate branch (which has already been merged). You will need to merge dev back into your branch to resolve this.